CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Unreleased
 
+## v0.21.1
+### Jun 13, 2025
+
+IMPROVEMENTS:
```

Comment:
You way want to call this out as a behavioral change.

Reply:
Or "BUG"